composer.json

```diff
@@ -18,8 +18,11 @@
         "elasticsearch/elasticsearch": "~1.3",
         "symfony-cmf/testing": "dev-master",
         "symfony/filesystem": "~2.4",
+        "symfony/console": "~2.5",
         "phpspec/prophecy-phpunit": "~1.0",
-        "matthiasnoback/symfony-dependency-injection-test": "0.7.1"
+        "matthiasnoback/symfony-dependency-injection-test": "0.7.1",
+        "doctrine/doctrine-bundle": "~1.3",
```

Comment:
DoctrineBundle is not required here... or ...coming to think of it I am missing the tag for the ORM subscriber.